webservices/docs.py

```diff
@@ -319,6 +319,10 @@
 
 SCHEDULE_A = SCHEDULE_A_TAG + '''
 
+The data is divided in two-year periods, called `transaction_year` this is derrived from
```

Comment:
Accounting for a couple of grammatical things here, should this instead say:

"The data is divided in two-year periods, called `transaction_year`, which is derived from"...